solvers.tex

```diff
@@ -0,0 +1,66 @@
+%% Solvers in SymPy
+
+
+SymPy has a solvers module to solve equations symbolically. There are two
+submodules to solve algebraic equations in SymPy, referred to as old solve
+i.e. \texttt{solve} and new solve i.e. \texttt{solveset}. Solveset is
+introduced with several design changes with respect to old \texttt{solve} to
+resolve the issues with old \texttt{solve}, for example old \texttt{solve}'s
+input API has many flags which are not needed and they make it hard for the
+user and the developers to work on solvers. In contrast to old solve, the
+\texttt{solveset} has a clean input API, It only asks for the much needed
+information from the user, following are the function signatures of old and new
+solve:
+
+\begin{verbatim}
+solve(f, *symbols, **flags)  # old solve
+solveset(f, symbol, domain)  # new solve
+\end{verbatim}
+
+The old \texttt{solve} has an inconsistent output API for various types of
+inputs, whereas the \texttt{solveset} has a canonical output API which is
+achieved using sets. It can consistently return various types of solutions.
+
+\begin{itemize}
+\item Single solution
+\end{itemize}
+\begin{verbatim}
+>>> solveset(x - 1)
+>>> {1}
+\end{verbatim}
+
+\begin{itemize}
+\item Finite set of solution, quadratic equation
+\end{itemize}
+\begin{verbatim}
+>>> solveset(x**2 - pi**2, x)
```

Comment:
I am yet to figure out, how to left align code inside itemize, so as to wrap all of them in one itemize.